internal/languages/language.go

```diff
@@ -0,0 +1,55 @@
+package languages
```

Comment:
It'd be good to add a README like @keegancsmith did when "vendoring in" the faster regexp/syntax logic. And maybe we can treat this the same as "vendoring", making sure to copy in the whole file? This makes future updates easier, as we can just copy-paste in the whole file.

Reply:
I think we can do that although my preference is to keep this as small as possible. It would pull in more changes to do the vendoring and those would be changes we would not use in zoekt.

Reply:
That works for me, I don't feel strongly about it. Could we still add a big file comment or README though, explaining what's going on?